src/mavsdk/core/mavlink_mission_transfer_server_test.cpp

```diff
@@ -413,3 +492,877 @@ TEST_F(MavlinkMissionTransferServerTest, ReceiveIncomingMissionCanBeCancelled)
     mmt.do_work();
     EXPECT_TRUE(mmt.is_idle());
 }
+
+TEST_F(MavlinkMissionTransferServerTest, SendOutgoingMissionEmptyMission)
```

Comment:
Most of these tests were copied from mavlink_mission_transfer_client_test.cpp, except for this one, which I wrote to explicitly test the ability to query empty missions.